fs/remote/blob.go

```diff
@@ -63,9 +59,8 @@ type Blob interface {
 	Size() int64
 	FetchedSize() int64
 	ReadAt(p []byte, offset int64, opts ...Option) (int, error)
-	Cache(offset int64, size int64, opts ...Option) error
 	Refresh(ctx context.Context, host source.RegistryHosts, refspec reference.Spec, desc ocispec.Descriptor) error
-	Close() error
+	Close()
```

Comment:
Why remove `error`?

Reply:
because there's no error to return after removing ``b.cache.Close()``.  See line 99 - line 105 of this file.